specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerAppsPatches.json

```diff
@@ -0,0 +1,566 @@
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsPatchNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Patch deletion has been started.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Patch does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "ContainerAppsPatches_Delete_0": {
+            "$ref": "./examples/ContainerAppsPatches_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/patches/{patchName}/skipConfig": {
+      "post": {
+        "tags": [
+          "ContainerAppsPatches"
+        ],
+        "operationId": "ContainerAppsPatches_SkipConfigure",
+        "description": "Configure the ContainerAppsPatch skip option by patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsPatchNameParameter"
+          },
+          {
+            "name": "patchSkipConfig",
+            "in": "body",
+            "required": true,
+            "description": "Configure patcher to skip a patch or not.",
+            "schema": {
+              "description": "Configure a patch to be skipped or not by passing a boolean.",
+              "type": "boolean"
+            }
```

Comment:
Your request schema should not be boolean because its not extensible. Please us "type": "object"